engine/runner/src/main/java/org/enso/runner/Main.java

```diff
@@ -418,11 +418,10 @@ private static Options buildOptions() {
             .build();
     var compileOption =
         cliOptionBuilder()
-            .hasArg(true)
-            .numberOfArgs(1)
-            .argName("package")
             .longOpt(COMPILE_OPTION)
-            .desc("Compile the provided package without executing it.")
+            .desc("Compile provided packages without executing.")
```

Comment:
- Can we do this change compatibly?
- If we keep the argument name and just remove the restriction on `numberOfArgs` it will be compatible
- ... but the name will stay in singular form